conf/base.config

```diff
@@ -9,15 +9,13 @@
  * run on the logged in environment.
  */
 
-process {
 
-  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-  memory = { check_max( 7.GB * task.attempt, 'memory' ) }
-  time = { check_max( 4.h * task.attempt, 'time' ) }
+process {
 
   errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
   maxRetries = 1
   maxErrors = '-1'
+  maxForks = 80
```

Comment:
Addition to prevent `AmazonEC2Exception`:

```console
com.amazonaws.services.ec2.model.AmazonEC2Exception: You have requested more vCPU capacity than your current vCPU limit of N allows for the instance bucket that the specified instance type belongs to.
```